examples/minimal-static-router/src/main.rs

```diff
@@ -0,0 +1,195 @@
+use route_rs_packets::EthernetFrame;
+use route_rs_runtime::link::{
+    primitive::{ClassifyLink, JoinLink, ProcessLink},
+    Link, LinkBuilder, PacketStream, ProcessLinkBuilder,
+};
+use route_rs_runtime::utils::test::{harness::run_link, packet_generators::immediate_stream};
+
+mod classifiers;
+mod processors;
+
+fn main() {
+    let data_v4: Vec<u8> = vec![
+        0xde, 0xad, 0xbe, 0xef, 0xff, 0xff, 1, 2, 3, 4, 5, 6, 8, 00, 0x45, 0, 0, 20, 0, 0, 0, 0,
+        64, 17, 0, 0, 192, 178, 128, 0, 10, 0, 0, 1,
+    ];
+    let data_v6: Vec<u8> = vec![
```

Comment:
nit: It isn't obvious what these bytes mean. Our examples could be a bit more descriptive if we can build up `EthernetFrame`s from an empty one.

Reply:
Good idea, I think a builder syntax or something is the next concept to support for Packets. I think doing that will allow us to have descriptive tests. Also Eric has his Pcap inputter done so we can leverage that as well in test cases.